webpack.config.example.js

```diff
@@ -61,5 +60,9 @@ module.exports = {
             query: 'brfs'
         }]
     },
-    devtool: 'cheap-source-map' // don't use eval-related options
+
+    // You may use any "devtool" except "eval" and "eval-source-map" due to
```

Comment:
this should be in the documentation, just wasted an hour trying to figure out why i couldn't get this working because this was missing!